vars/buildPlugin.groovy

```diff
@@ -66,6 +66,17 @@ def call(Map params = [:]) {
                             isMaven = fileExists('pom.xml')
                             incrementals = fileExists('.mvn/extensions.xml') &&
                                     readFile('.mvn/extensions.xml').contains('git-changelist-maven-extension')
+                            if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
+                                if (isUnix()) {
+                                    sh(script: 'git clean -xffd || true',
+                                       label:'Clean for incrementals',
+                                       returnStatus: true) // Ignore failure if CLI git is not available
```

Comment:
Does the `|| true` not sufice?

Reply:
It should, but I was practicing an "abundance of caution" in hopes that a peripheral cleanup operation would have even less chance of inadvertently breaking a build.

Reply:
So then `returnStatus: true` should suffice (both both platforms)?

Reply:
Yes, should. Same "abundance of caution" logic.  If ignoring the return value inside the script somehow did not work, also take the active measure of ignoring the return value from the step.  Both [`sh`](https://www.jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script)` and [`bat`](https://www.jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#bat-windows-batch-script) step documentation say:

> Normally, a script which exits with a nonzero status code will cause the step to fail with an exception.  If this option is checked, the return value of the step will instead be the status code.

Reply:
I've changed to hide the output from git clean and ignore the return code in c05194b38dddced80f031ed365ca9274950ab7dc .  Let me know if other changes are needed.